DisplaySettings/DisplaySettings.json

```diff
@@ -1970,8 +2145,8 @@
                 "$ref": "#/definitions/result"
             }
         },
-        "setVolumeLeveller":{
-            "summary": "(Version 2) Sets the Volume Leveller level. Volume Leveler is an advanced volume-control solution that maintains consistent playback levels for content from different sources.",
+        "setVolumeLeveller(v2)":{
```

Comment:
Remove `(v2)` from the method signature. Application calls that include `(v2)` in the method signature will fail.

Reply:
As discussed keeping only version2 methods and removed (v2) from method.